pkg/operator/workloads/workflow.go

```diff
@@ -32,6 +32,9 @@ import (
 	"github.com/cortexlabs/cortex/pkg/operator/config"
 )
 
+var cortexCPUReserve = kresource.MustParse("800m")   // FluentD (200), Nvidia (100), StatsD (100), Kube Proxy, (100) Node capacity - Node availability 300 CPU
```

Comment:
Would it be relatively easy to only count Nvidia for GPU nodes? Or not worth it?